nginx.conf

```diff
@@ -1,12 +1,27 @@
+worker_processes auto;
+
+worker_rlimit_nofile 100000;
+
 events {
-  worker_connections 1024;
+  worker_connections 4000;
 }
 
 http {
-
+  
   include mime.types;
   default_type application/octet-stream;
   sendfile on;
+  tcp_nopush on;
+  tcp_nodelay on;
+  server_tokens off;
+  client_max_body_size 100M;
+  
+  access_log off;
+  
+  reset_timedout_connection on;
+  client_body_timeout 10;
+  send_timeout 2;
+  keepalive_timeout 30;
```

Comment:
default timeouts are much higher, >= 60s. Changing timeouts to values you're suggesting can break many applications using this image. 

I would like to keep all timeouts on default levels, both in http and in server block, since these settings have to be adjusted to individual applications.